book-blurbs/drip-summary-300-words.md

```diff
@@ -0,0 +1,26 @@
+_Development Research in Practice_ is intended to teach all users of development data 
+how to handle data effectively, efficiently, and ethically. 
+The Handbook leads the reader through a complete empirical research project,
+providing links to continuously-updated resources on the DIME Wiki
+as well as illustrative examples from 
+the Demand for Safe Spaces impact evaluation conducted in Rio de Janeiro. 
+The premise is that standardizing and simplifying data tasks 
+improves research quality and enables more effective collaboration. 
+The first chapter introduces principles for conducting reproducible, 
+transparent, and credible development research. 
+The second chapter describes technical tools and set-up necessary
+for effective research in today’s collaborative and data-driven world. 
+The third chapter details the theory and practice of data-driven measurement frameworks,
+including research design, data mapping, and implementation of randomized study designs. 
+The fourth chapter covers the data acquisition process for primary and secondary data,
+and details special considerations for collecting high-quality survey data. 
+The fifth chapter dives into the process of cleaning and exploring research data. 
+The sixth chapter provides a guide to building analytical datasets, 
+writing data analysis code, and creating reproducible research outputs. 
+The seventh chapter addresses research publication, 
+including the preparation of reproducibility packages and publishing data. 
+The practical approach and narrative structure of these chapters are designed
+for both newcomers to empirical researchas well as experienced professionals. 
+The Development Research in Practice handbook is therefore a quintessential desk reference 
+for anyone who handles data as part of a research team, policymakers and managers who oversee such endeavors, 
```

Comment:
```suggestion
for anyone who handles development data as part of a research team, policymakers and managers who oversee such endeavors, 
```
We make the distinction in the beginning, but I'd be more comfortable if we make it clear we are abundantly clear that the scope is limited to development data